definitions/infra-awselasticsearchcluster/summary_metrics.yml

```diff
@@ -3,14 +3,14 @@ providerAccountName:
     key: providerAccountName
   title: AWS Account
   unit: STRING
-indexingRate:
+indexingRateReqsMin:
   query:
     eventId: entity.guid
     select: (average(aws.es.IndexingRate.byCluster)) / 60
```

Comment:
This queries in SummaryMetric are different than GoldenMetric  `/60`

Reply:
We will add a new unti REQUEST_PER_MINUTE in the future to align this 2 GM and SM. We can proceed with the PR :)